TypeCobol/Compiler/Diagnostics/TypeCobolChecker.cs

```diff
@@ -253,13 +253,13 @@ public static void OnNode(Node node)
                     var actual = call.Arguments[c].StorageAreaOrValue;
                     if (actual.IsLiteral) continue; //TODO
 
-                    var callArgName = actual.StorageArea.ToString();
                     var found = node.GetDataDefinitionFromStorageAreaDictionary(actual.StorageArea);
-                   if (found == null)
+                    if (found == null)
                     {
                         continue;
                     }
 
+                    var callArgName = actual.StorageArea.ToString();
```

Comment:
Ok with this fix, but we also need to check that `searchedStorageArea` is not null within `GetDataDefinitionFromStorageAreaDictionary`.

Reply:
done
https://github.com/TypeCobolTeam/TypeCobol/pull/1573/commits/ada299ba51eb65b02b486e60ab6de3e72c064bcb